docker-compose.yml

```diff
@@ -51,14 +51,14 @@ services:
       retries: 3
     command: bigchaindb -l DEBUG start
   tendermint:
-    image: tendermint/tendermint:0.12
+    image: tendermint/tendermint:0.19.2
     volumes:
       - ./compose/tendermint/tmdata/config.toml:/tendermint/config.toml
     entrypoint: ''
     ports:
       - "46656"
       - "46657"
-    command: bash -c "tendermint init && tendermint node"
+    command: sh -c "tendermint init && tendermint node --proxy_app=tcp://bigchaindb:46658"
```

Comment:
@codegeschrei You can also [disable empty blocks](https://github.com/bigchaindb/bigchaindb/blob/master/docker-compose.yml#L56)

Reply:
oh right! good idea! and thanks for the link